internal/expr/prepare.go

```diff
@@ -77,12 +204,30 @@ func (pe *ParsedExpr) Prepare(args ...any) (expr *PreparedExpr, err error) {
 			sql.WriteString("?")
 			ins = append(ins, p)
 		case *outputPart:
+			outCols, err := prepareOutput(ti, p)
+			if err != nil {
+				return nil, err
+			}
+			startCol := n
+			for i, c := range outCols {
+				sql.WriteString(c.String())
+				sql.WriteString(" AS _sqlair_")
+				sql.WriteString(c.name)
```

Comment:
Per conversation, I think we can use just `_sqlair_1`, without the name, and then have a private slice that maps from those indexes into the fields.

Reply:
Changed.